user-guide/Advanced_Modules/toc.yml

```diff
@@ -826,6 +826,8 @@ items:
         topicUid: DomHelper_class
       - name: DOM events
         topicUid: DOM_events
+      - name: DOM multiple sections
```

Comment:
I'm not entirely sure if this is the best location for this page. It's a detail about sections, so it's not something I'd expect to find between much more central concepts like DOM events and history. I'm afraid this might lead to some confusion. Could we move this elsewhere? Possibly it could be added under "SectionDefinition object" or added as a section on [that page](https://docs.dataminer.services/user-guide/Advanced_Modules/DOM/DOM_objects/DOM_SectionDefinition.html)?

Reply:
I moved it under the DomInstance page after consulting with @Skyline-ThomasGH